com.cinecred.cinecred.yml

```diff
@@ -0,0 +1,43 @@
+id: com.cinecred.cinecred
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+command: cinecred
+finish-args:
+  - --share=ipc
+  - --share=network
```

Comment:
why do you need network access ?

Reply:
There's optional integration with Google Sheets: https://cinecred.com/guide/project-folder/#creating-a-project